component/attr_cache/attr_cache.go

```diff
@@ -569,12 +579,17 @@ func NewAttrCacheComponent() internal.Component {
 // On init register this component to pipeline and supply your constructor
 func init() {
 	internal.AddComponent(compName, NewAttrCacheComponent)
+	
 	attrCacheTimeout := config.AddUint32Flag("attr-cache-timeout", defaultAttrCacheTimeout, "attribute cache timeout")
 	config.BindPFlag(compName+".timeout-sec", attrCacheTimeout)
+	
 	noSymlinks := config.AddBoolFlag("no-symlinks", false, "whether or not symlinks should be supported")
 	config.BindPFlag(compName+".no-symlinks", noSymlinks)
 
 	cacheOnList := config.AddBoolFlag("cache-on-list", true, "Cache attributes on listing.")
 	config.BindPFlag(compName+".cache-on-list", cacheOnList)
 	cacheOnList.Hidden = true
+	
+	maxTotalFiles := config.AddUint32Flag("max-total-files", defaultMaxTotalFiles, "Max total file attributes in cache.")
+	config.BindPFlag(compName+".max-total-files", maxTotalFiles)
```

Comment:
This part we can skip, this basically creates a command line parameter for this flag. I guess not many customers will go for this and based on that we can skip creating cli parameter for this field, just in the config it shall be fine.